tools/ports/sdl2_mixer.py

```diff
@@ -30,11 +30,28 @@ def create():
     shutil.copytree(source_path, dest_path)
 
     final = os.path.join(dest_path, libname)
-    ports.build_port(dest_path, final, [], ['-DOGG_MUSIC', '-O2', '-s', 'USE_VORBIS=1', '-s', 'USE_SDL=2'],
-                     ['dynamic_flac', 'dynamic_fluidsynth', 'dynamic_mod', 'dynamic_modplug', 'dynamic_mp3',
-                      'fluidsynth', 'load_mp3', 'music_cmd', 'music_flac', 'music_mad', 'music_mod',
-                      'music_modplug', 'playmus.c', 'playwave.c'],
```

Comment:
what is the impact of removing `dynamic_modplug` and the other stuff removed here?

Reply:
Those files are wrapped in
```
#ifdef MUSIC_MOD_MODPLUG
```
and similars, so they add nothing unless the macros are defined.

Note that most of those files were renamed ~ `dynamic_modplug` was renamed to `music_modplug`